CHANGES

```diff
@@ -110,6 +101,11 @@ Notations
 
 Tools
 
+- New warnings system for coqtop/coqc. Can be configured with the flag "-w" or
+  with the Vernacular command "Set Warnings".
+- Warnings are now printed by coqc.
+- In asynchronous mode, Coq is now capable of recovering from errors and
+  continue processing the document.
 - coqc accepts a -o option to specify the output file name
```

Comment:
For what of changes you mean? For the warnings.

Not really internal, right? That's right that it is visible. So, maybe "tools" is indeed the best approximation if one considers categories already used in the CHANGES file (even if, to tell the truth, I hardly see "coqtop" and "coqc" as tools, in contrast to "coqdep", "coqdoc", "coq_makefile", ...)

Otherwise, maybe "general infrastructure", since the warning infrastructure is really independent in itself of "coqc", "coqtop".